src/components/tipRecords/AuthorAndDate.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div
+    class="author-and-date"
+    :title="address"
+    @click.stop
+  >
+    <router-link
+      :to="{
+        name: 'user-profile',
+        params: {
+          address,
+        },
+      }"
+    >
+      <Avatar :address="address" />
+      <div class="author-name">
+        <span
+          v-if="chainName"
+          class="chain-name"
+        >
+          {{ chainName }}
+        </span>
+        <span
+          v-if="!chainName || (tipRecord && chainName)"
```

Comment:
> Let's ask designers about the possibility to make this component looks the same in all cases. I don't see reasons to show it differently

cc @onvisions

Reply:
First of all which component exactly? Where I can visually see it (I'm not coding it so it will be useful to have a screenshot of what you are talking about when there are questions for me)? If I understand right your discussion above is about the suggestion to **avoid including linked blocks within other linked blocks** (in the tip item).
However there is a reason why things were designed this way. It's following the UX/UI pattern of Twitter feed items and I consider it more intuitive navigation from user's perspective. Even now implementation has some problem as the **fetched preview** according to my understanding should lead to the URL being fetched. Now it's leading nowhere. It's not active link although the cursor is set to pointer. I'll visualize it here to make my point:
![Artboard – 1](https://user-images.githubusercontent.com/12225410/91421194-4a56e500-e85e-11ea-953f-fb685d2cea09.png)

One more thing to remind. We have approved designs for rich content previews for presenting different types of feed items according to their content (audio, video, just text, just images, direct tips etc.). They were published to Aeternity forum, received feedback and approval and as far as I know their implementation will be part of the coming features to be worked on. I'm reminding this because rich content previews will require some change in the feed items views and their components I guess... If we have to further discuss this issues we may have syncing call may be?

Reply:
I have just checked Twitter and got the idea 👌

Reply:
I have created an [issue](https://github.com/aeternity/superhero-ui/issues/779) for this, we can continue our discussion there.